README.md

```diff
@@ -4,9 +4,29 @@ UnifiedPush provider for Nextcloud - android application
 [<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png"
      alt="Get it on F-Droid"
      height="80">](https://f-droid.org/packages/org.unifiedpush.distributor.nextpush/)
+## Requirements
+**Nextcloud Server**
+
+[Server App Install](https://github.com/UP-NextPush/server-app)
+
+**Android Apps**
+
+[Nextcloud Application](https://f-droid.org/packages/com.nextcloud.client/)
+
+[NextPush Client](https://f-droid.org/en/packages/org.unifiedpush.distributor.nextpush/)
+
+[Unified Push Client](https://unifiedpush.org/users/apps/)
+
+[Unified Push Test Client](https://f-droid.org/en/packages/org.unifiedpush.example/) - For testing purposes only. Not required for operation.
 
 ## Usage
-This application require the [server application](https://github.com/UP-NextPush/server-app) to be installed on the server and the [Nextcloud application](https://apps.nextcloud.com/apps/android_nextcloud_app) on the mobile phone.
+
+1. Install and sign into your Nextcloud account using the official Nextcloud sync client.
+2. Install the NextPush client and sign into you Nextcloud account.
+3. Install one of the supported Unified Push clients with the following url.
+`Please provide an example of the URL you are to input into the Unified Push Client`
```

Comment:
I don't know what you mean with this. It is related to the client. For instance with fedilab it would be the mastodon/fediverse URL and for fluffychat the matrix server URL.